test/test-zero-len-file.sh

```diff
@@ -50,7 +50,8 @@ begin_test "pull zero len file"
   clone_repo "$reponame" clone
   rm clone.log
 
-  git status | grep "working directory clean"
+  git status 2>&1 | tee status.log
+  grep "working (directory|tree) clean" status.log
```

Comment:
Definitely need `egrep` (or `grep -E`, same thing) for this kind of RE. Does this need to go to a log file first? It seems the direct pipe was sufficient before, why change it?